tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = py36, lint
+skip_missing_interpreters = True
+
+[travis]
+python =
+  3.6: py36, lint
+
+[testenv]
+commands =
+  pytest -v --timeout=30 --cov=imjoy --cov-report= {posargs}
+deps =
+  -rrequirements.txt
+  -rrequirements_test.txt
+
+[testenv:lint]
+basepython = python3
+ignore_errors = True
+commands =
+  black --check ./
+deps =
+  -rrequirements_lint.txt
```

Comment:
is there a space between -r and requirements_lint.txt?

Reply:
No. This is according to docs:
https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints

Reply:
I see